CHANGES.rst

```diff
@@ -8,11 +8,26 @@ Unreleased
 
 * ``config_interpolation`` parameter for :class:`.Reddit` supporting basic and
   extended modes.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` contains a new parameter,
+  ``fetch``, that toggles the automatic fetching of existing data from Reddit.
+  It is set to true on default.
```

Comment:
As a point of style, I'd like Boolean values to be given as Python keywords (`True`/`False`) with capitalization and monospacing.